src/main/java/com/featureprobe/sdk/server/model/PredicateType.java

```diff
@@ -0,0 +1,48 @@
+package com.featureprobe.sdk.server.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum PredicateType {
+    IS_ONE_OF,
+    ENDS_WITH,
+    STARTS_WITH,
+    CONTAINS,
+    MATCHES_REGEX,
+
+    IS_NOT_ANY_OF,
+    DOES_NOT_END_WITH,
+    DOES_NOT_START_WITH,
+    DOES_NOT_CONTAIN,
+    DOES_NOT_MATCH_REGEX;
+
+    private static final Map<String, PredicateType> namesMap = new HashMap<>();
```

Comment:
Add `value` attribute to this enum. `Hashmap` can be used without it.

For example, define a type:
`IS_ONE_OF("is one of")`

And add the `getValue` (just return this.value) function to current enum.

Reply:
Yeah that's what I want to do! I was confused whether to use this or follow the style in `enum ConditionType`, so is it okay to change that enum too?

https://github.com/FeatureProbe/server-sdk-java/blob/8bbd3e3794c146a4d2fcb7568100da3d47bca416/src/main/java/com/featureprobe/sdk/server/model/ConditionType.java#L12-L18